daemon/daemon_windows.go

```diff
@@ -191,8 +191,11 @@ func verifyDaemonSettings(config *config.Config) error {
 func checkSystem() error {
 	// Validate the OS version. Note that dockerd.exe must be manifested for this
 	// call to return the correct version.
-	if osversion.Get().MajorVersion < 10 || osversion.Build() < osversion.RS5 {
-		return fmt.Errorf("this version of Windows does not support the docker daemon (Windows build %d or higher is required)", osversion.RS5)
+	if osver := osversion.Get(); osver.MajorVersion < 10 || osver.Build < osversion.RS5 {
+		if osver.Build == 9200 { // TODO implement a proper check / constant for this build number? https://github.com/microsoft/hcsshim/issues/1080#issuecomment-1281601474
+			return fmt.Errorf("this build is not 'manifested', which breaks many Windows APIs (see https://github.com/golang/go/issues/44466 for a list of some)")
```

Comment:
From the linked Go ticket, it _seems_ most things "work", but may not be optimised. Thinking;

a. should this be (big fat) WARNING?
b. or should we have a `DOCKER_SKIP_VERSION_CHECK` env-var (we had something like that for Linux kernel versions)

Thinking; if it's useful to be able to run the binary (for testing purposes) without manifesting 🤔  (Although I guess our build scripts do the manifesting, so maybe it's not correct to allow overriding)

Reply:
As to `build 9200` - do we know if that was an actual (public) build? Or was this an internal build that was picked (as in: this would never be found "in the wild")?

Reply:
At least opened a PR to propose something to make it more discoverable 😅 https://github.com/microsoft/hcsshim/pull/1546

Reply:
Should we go with this for now to improve the micro-experience while they deliberate on a better solution?  or would you rather we close this and wait for something better instead?